src/index.html

```diff
@@ -1,13 +1,321 @@
+            </li>
+            <li class="nav__item">
+              <a href="#contact" class="nav__link ">Contacts</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a
+          href="tel:+1 234 5555-55-55"
+          class="menu__phone-number"
+        >
+          +1 234 5555-55-55
+        </a>
+        <a
+          href="tel:+1 234 5555-55-55"
+          class="menu__book-ride"
+        >
+          Book a test ride
+        </a>
+      </div>
+    </aside>
+    <!-- #endregion -->
+
+    <!-- #region main -->
+    <main class="main-content page__main-content">
+      <div class="main-content__container">
+        <section class="main-content__move-free move-free" id="move-free">
+          <h2 class="move-free__title move-free__title--align-left">Move Free</h2>
+          <p class="move-free__description">
+            Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+          </p>
+        </section>
+
+        <section class="main-content__compare-bikes compare-bikes" id="compare-bikes">
+          <h2 class="compare-bikes__title">Compare Bikes</h2>
+
+          <div class="compare-bikes__content">
+            <article class="compare-bikes__bike-card bike-card">
+              <a href="#" class="bike-card__link">
+                <img
+                  src="./images/bikes/sporty-4.png"
+                  alt="Sporty 4 Bike"
+                  class="bike-card__image"
+                >
+              </a>
+              <h3 class="bike-card__title bike-card__bold-text">Sporty 4</h3>
+              <p class="bike-card__description">
+                The iconic frame brought to a new performance height as a sporty, active ride.
+              </p>
+              <p class="bike-card__price bike-card__bold-text">$ 2 590</p>
+            </article>
+
+            <article class="compare-bikes__bike-card bike-card">
+              <a href="#" class="bike-card__link">
+                <img
+                  src="./images/bikes/ride-in-town-st.png"
+                  alt="Ride in town ST Bike"
+                  class="bike-card__image"
+                >
+              </a>
+              <h3 class="bike-card__title bike-card__bold-text">Sporty 4</h3>
+              <p class="bike-card__description">
+                An open frame for an upright riding position as the most comfortable ride in town.
+              </p>
+              <p class="bike-card__price bike-card__bold-text">$ 2 590</p>
+            </article>
+
+            <article class="compare-bikes__bike-card bike-card">
+              <a href="#" class="bike-card__link">
+                <img
+                  src="./images/bikes/agile-ride-3.png"
+                  alt="Agile ride 3 Bike"
+                  class="bike-card__image"
+                >
+              </a>
+              <h3 class="bike-card__title bike-card__bold-text">Sporty 4</h3>
```

Comment:
Add empty line between this blocks and all the similar cases where you have two or more first child blocks of one and same parent block